applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -64,40 +63,37 @@ static uint8_t csip_rsi_adv_data[BT_CSIP_RSI_SIZE];
 
 static uint8_t flags_adv_data[] = {BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR};
 
-static uint8_t gap_appear_adv_data[] = {
-	BT_BYTES_LIST_LE16(BT_APPEARANCE_WEARABLE_AUDIO_DEVICE_EARBUD)};
+static uint8_t gap_appear_adv_data[] = {BT_BYTES_LIST_LE16(CONFIG_BT_DEVICE_APPEARANCE)};
 
 static const uint8_t cap_adv_data[] = {
 	BT_UUID_16_ENCODE(BT_UUID_CAS_VAL),
 	BT_AUDIO_UNICAST_ANNOUNCEMENT_TARGETED,
 };
 
-static uint8_t tmap_adv_data[] = {
-	BT_UUID_16_ENCODE(BT_UUID_TMAS_VAL),
-	BT_BYTES_LIST_LE16(BT_TMAP_ROLE_UMR | BT_TMAP_ROLE_CT),
-};
-
 static uint8_t csis_rsi_adv_data[BT_CSIP_RSI_SIZE];
 
 #if defined(CONFIG_BT_AUDIO_TX)
 #define AVAILABLE_SOURCE_CONTEXT                                                                   \
-	(BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL |                \
-	 BT_AUDIO_CONTEXT_TYPE_MEDIA)
+	(BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL)
 #else
 #define AVAILABLE_SOURCE_CONTEXT 0x0000
-#endif
+#endif /* CONFIG_BT_AUDIO_TX */
 
 #if defined(CONFIG_BT_AUDIO_RX)
-#define AVAILABLE_SINK_CONTEXT (BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | BT_AUDIO_CONTEXT_TYPE_MEDIA)
+#define AVAILABLE_SINK_CONTEXT                                                                     \
+	(BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | BT_AUDIO_CONTEXT_TYPE_MEDIA |                         \
+	 BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL)
 #else
 #define AVAILABLE_SINK_CONTEXT 0x0000
-#endif
+#endif /* CONFIG_BT_AUDIO_RX */
 
 static uint8_t unicast_server_adv_data[] = {
 	BT_UUID_16_ENCODE(BT_UUID_ASCS_VAL),
 	BT_AUDIO_UNICAST_ANNOUNCEMENT_TARGETED,
 	BT_BYTES_LIST_LE16(AVAILABLE_SINK_CONTEXT),
+#if defined(CONFIG_BT_AUDIO_TX)
```

Comment:
Is this necessary when we set AVAILABLE_SOURCE_CONTEXT 0x0000 on line 79? This is also guarded by CONFIG_BT_AUDIO_TX